### DISCUSSION TITLE
Welcome to Kimai 2 discussions 👋

### DISCUSSION THREAD

**Discussion by kevinpapst** [kimai/kimai#2171] [General]
## Welcome! 
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about ❓ 
  * Share ideas 💡
  * Engage with other community members 🗣️ 
  * Welcome others and are open-minded, remember that this is a community we build together 💪

To get started, comment below with an introduction of yourself and tell us about what you do within this community.

**Comment by vic-t** [comment]
Should be a sticky. :grin: Not sure that's possible with Github, though.
